manifests/bind_event.pp

```diff
@@ -0,0 +1,34 @@
+# Define: qpid::bind_event
+#
+# This define binds the event queue in katello with the correct messages from candlepin to filter out uneeded messages
+#
+# === Parameters
+#
+# $hostname::                   FQDN of the host
+#
+# $ss_cert::                    SSL cert to use for qpid-config commands
+#
+# $candlepin_event_queue::      Name of the Candlepin event queue
+#
+# $title::                      Title of event
+define qpid::bind_event(
+include qpid::params
+  String $hostname = $::fqdn,
```

Comment:
Looks like you're reading the Puppet 4 docs, at the moment don't use types like Strings on the variables, it's not supported in Puppet 3